widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css

```diff
@@ -4515,17 +4515,42 @@ div.dijitDateTextBox.dijitComboBox {
   outline-offset: var(--focus-ring-offset);
 }
 
+.fileBrowserColumn > .body {
+  display: flex;
+  flex-direction: column;
+  flex: none;
+}
+.fileBrowserColumn > .body > .folder {
+  width: max-content;
+  min-width: 100%;
+}
```

Comment:
NOTE: These and following rules fix the selection rectangle of folder tree nodes not stretching till the end of line.